profiles/SampleEmpath-setup.yaml

```diff
@@ -0,0 +1,695 @@
+  :name: handbook
+  :is_leather: true
+  :hinders_lockpicking: false
+- :adjective: kertig 
+  :name: maul
+  :hinders_lockpicking: false
+- :adjective:  diamondique
+  :name: hhr'ata
+  :hinders_lockpicking: false
+- :adjective:  slender
+  :name: sledgehammer
+  :hinders_lockpicking: false
+  :is_leather: true
+- :adjective: dueling
+  :name: iltesh
+  :hinders_lockpicking: false
+- :adjective: agonite
+  :name: spatha
+  :hinders_lockpicking: false
+- :adjective: diacan
+  :name: trident
+  :hinders_lockpicking: false
+- :adjective: nisha
+  :name: shortbow
+  :is_leather: true
+  :hinders_lockpicking: false
+- :adjective: forester's
+  :name: stonebow
+  :is_leather: true
+  :hinders_lockpicking: false
+- :adjective: diamondwood
+  :name: quarterstaff
+  :hinders_lockpicking: false
+- :adjective: firesilk
+  :name: sling
+  :is_leather: true
+  :hinders_lockpicking: false
+- :adjective: tyrium
+  :name: nodachi
+  :is_leather: false
+  :hinders_lockpicking: false 
+- :adjective: belaying
+  :name: pin
+  :is_leather: true
+  :hinders_lockpicking: false
+##tools
+- :name: rasp
+  :tie_to: shaper.toolbelt
+- :name: shaper
+  :tie_to: shaper.toolbelt
+- :name: drawknife
+  :tie_to: shaper.toolbelt
+- :name: carving knife
+  :tie_to: shaper.toolbelt
+- :name: burin
+  :is_leather: true
+- :name: brazier
+- :name: loop
+  :is_leather: true
+- :name: rod
+  :adjective: imbuement
+  :is_leather: true  
+  
+
+#anything that is to be worn must be listed below - armor or weapons
+gear_sets:
+  #default set to be used in/out of combat
+  standard:  
+  - demonscale gloves
+  - quilted hood
+  - embroidered pants
+  - titanese shirt
+  - pot lid
+  - elbow wraps
+  - iron bracer
+  - hand claws
+  - knee spikes
+  #used by ;bescort to make swimming go easier - remove body pieces
+  swimming:
+  - elbow wraps
+  - iron bracer
+  - hand claws
+  - knee spikes  
+  #used by scripts such as ;rdc, ;udc, or ;steal
+  stealing:
+  - elbow wraps
+  - iron bracer
+  - hand claws
+  - knee spikes
+
+#list of scripts that will block ;sanowret-crystal from draining your concentration
+sanowret_no_use_scripts:
+- sew
+- carve
+- tinker
+- forge
+- remedy
+- shape
+- enchant
+- outdoorsmanship
+- combat-trainer
+- buff  
+
+#use only if you wear a biomechanical tarantula
+#these are very basic settings - will need customzing
+# tarantula_noun: tarantula
+# tarantula:
+#   Armor:
+#     combat:
+#     - Plate Armor
+#     - Shield Usage
+#   Weapon:
+#     combat:
+#     - Melee Mastery
+#     - Missile Mastery
+#   Magic:
+#     combat:
+#     - Life Magic
+#     non_combat:
+#   Survival:
+#     combat:
+#     - Evasion
+#     - Skinning
+#     non_combat:
+#     - Athletics
+#   Lore:
+#     combat:
+#     - Tactics    
+#     non_combat:
+#     - Appraisal
+#     - Performance
+
+# #skills to never ever "eat"
+# tarantula_excluded_skills:
+#   - Empathy
+
+# don't say anything when entering a hunting room
+empty_hunting_room_messages:
+- /   
```

Comment:
❔ Does this say `/` in your room?

Reply:
No, I use that too.  It does this:

>'/
>
You must specifiy an emote.

Reply:
Many of us use this to avoid the telltale "anyone here?"  It's definitely safe to use.

Reply:
Makes sense, thanks